CHANGELOG.next.asciidoc

```diff
@@ -11,7 +11,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 *Affecting all Beats*
 
 * Upgrade to Go 1.18. Certificates signed with SHA-1 are now rejected. See the Go 1.18 https://tip.golang.org/doc/go1.18#sha1[release notes] for details. {pull}32493[32493]
-
+* Add lock path when returning ErrAlreadyLocked. {pull}[32980]
```

Comment:
I believe it does not make sense to mention `ErrAlreadyLocked` on this changelog. If it was the `CHANGELOG-developer.next.asciidoc` it would be ok.

Maybe rephrase to something like:
> Add lock path to error message when the lock file already exists.

Reply:
Also the format for the link at the end should be: `{pull}32980[32980]`